addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -347,5 +364,8 @@ odoo.define('website_slides.quiz', function (require) {
         },
     });
 
-    return Quiz;
+    return {
+        Quiz: Quiz,
+        websiteSlidesQuizNoFullscreen: sAnimations.registry.websiteSlidesQuizNoFullscreen
```

Comment:
why do you need to return it ? As to extend it, you can do `sAnimations.registry.websiteSlidesQuizNoFullscreen`

Reply:
Indeed but I'm ok with the fact that this 'defines' 2 things, so it probably should return everything it defines.
Plus we already use that in other files currently (like 'slides_course_join.js').

What do you think @jem-odoo ?